### ISSUE TITLE
InfraIntegrationsMatch assigns a Failure to status to those who are missing a definition file

### ISSUE THREAD

**Issue by uturunku1** [newrelic/newrelic-diagnostics-cli#2]
## Description

This InfraIntegrationsMatch is giving a false positive that says that a customer is missing a definition file when they are actually not because new versions of the agent do not require it. 

Source
https://docs.newrelic.com/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integration-configuration-overview :

>Standard: This is the format used by most on-host integrations. This configuration uses two files: a definition file and a configuration file. For more details, see Standard configuration.
Newer: Starting December 2019, infrastructure agent version 1.8.0 began supporting a new format used by some integrations. This format uses a single configuration file and provides other improvements. For more details, see Newer configuration. 

## Steps to Reproduce
Run `./nrdiag -suites infra` in an environment that has the infra agent installed, a version newer than 1.8.0

You'll get the error:

```
Found matching integration files with some errors: Configuration file 'C:\Program Files\New Relic\newrelic-infra\integrations.d\perfmon-config.yml' does not have matching Definition file Definition file 'C:\Program Files\New Relic\newrelic-infra\custom-integrations\nri-perfmon-definition.yml' does not have matching Configuration file
```
## Expected Behavior

This task should not fail for customers that have a newer version of the infra agent. A possible fix to this bug could be to add a check for looking at what version are they are using, and then based on that check we can judge if they are missing a definition file or not.

**Comment by uturunku1** [comment]
https://github.com/newrelic/newrelic-diagnostics-cli/pull/46
